gvproxy.hcl

```diff
@@ -0,0 +1,50 @@
+description = "gvproxy is a new network stack based on gVisor."
```

Comment:
This package seems good though, happy to add it.

Reply:
ok. will split the package off. and change the podman install to not depend of gvproxy and generate a setup script instead for gvproxy (rather than force the config). sound good?

Reply:
I must admit I don't fully understand the relationship between the two. Does podman _require_ gvproxy? Or is it an optional thing?

Either way, I would suggest those setup scripts should go in the `gvproxy` package, not podman.

Reply:
Podman is hard coded to use gvproxy. It cannot run without it and, when installing podman with brew, it automatically installs gvproxy as a dependency. I think having this as a dependency is fine. Theoretically, podman could use other VM proxy tools than gvproxy, but for now it's the only tool it uses. The result is quite nice - you can install podman with a single hermit install command and then build and run docker containers entirely in user space! No other installation required!